spec/index.bs

```diff
@@ -1255,6 +1255,34 @@ For more information on issuing CORS requests for image and video elements, cons
 - [[html#the-img-element]] <{img}>
 - [[html#media-elements]] {{HTMLMediaElement}}
 
+## Color Spaces ## {#color-spaces}
+
+WebGPU does not provide color management. All values within WebGPU (such as texture elements)
+are raw numeric values, not color-managed color values.
+
+WebGPU *does* interface with color-managed outputs (via {{GPUCanvasContext}}) and inputs
+(via {{GPUQueue/copyExternalImageToTexture()}}).
+Color conversion must be performed between the WebGPU numeric values and the external color values.
+These interface points locally define a color space, in which the WebGPU numeric values are to be
+interpreted, from options defined in [[css-color-4#predefined]].
+
+<script type=idl>
+enum GPUPredefinedColorSpace {
+    "srgb",
+};
+</script>
+
+Issue: Replace this with the `PredefinedColorSpaceEnum` from the
+[canvas color space proposal](https://github.com/WICG/canvas-color-space/blob/main/CanvasColorSpaceProposal.md).
+
+Issue: The most urgent additional option to add here is "srgb-linear", which is needed to directly
+create linearly-encoded textures (e.g. create a linearly-encoded float16 texture from an sRGB PNG).
+
+<dl dfn-type=enum-value dfn-for=GPUPredefinedColorSpace>
+    : <dfn>"srgb"</dfn>
+    :: The CSS predefined color space <a value for=color()>srgb</a>.
```

Comment:
"CSS"?

Reply:
The link (`<a value for=color()>srgb</a>`) takes you to the [appropriate definition](https://drafts.csswg.org/css-color-4/#valdef-color-srgb), I don't think "CSS" needs to be clarified here.